src/ui/events.js

```diff
@@ -23,16 +23,16 @@ import type LngLat from '../geo/lng_lat';
  */
 export class MapMouseEvent extends Event {
     /**
-     * The event type (one of [`mousedown`](#map.event:mousedown),
-     * [`mouseup`](#map.event:mouseup),
-     * [`click`](#map.event:click),
-     * [`dblclick`](#map.event:dblclick),
-     * [`mousemove`](#map.event:mousemove),
-     * [`mouseover`](#map.event:mouseover),
-     * [`mouseenter`](#map.event:mouseenter),
-     * [`mouseleave`](#map.event:mouseleave),
-     * [`mouseout`](#map.event:mouseout),
-     * [`contextmenu`](#map.event:contextmenu)).
+     * The event type (one of {@link Map.event:mousedown},
+     * {@link Map.event:mouseup},
```

Comment:
Is there a different format for these links that would make it easier to fix the links in the generated docs?

Reply:
@asheemmamoowala https://github.com/mapbox/mapbox-gl-js-docs/pull/272 includes a new function to handle and route JSDoc `@link` and so it will fix https://github.com/mapbox/mapbox-gl-js-docs/pull/191 🎉